src/main/java/uk/gov/ccs/swagger/sso/api/OrganisationUserApi.java

```diff
@@ -133,7 +133,7 @@ private com.squareup.okhttp.Call organisationsOrganisationIdUsersGetValidateBefo
 
     /**
      * Allows a user to retrieve users for a given organisation
-     * NOTE:- query params page-size, current-page  Sample request:                    GET /organisations/1/users?page-size&#x3D;10,current-page&#x3D;1
+     * NOTE:- query params page-size, current-page  Sample request:                    GET /organisation-profile1/users?page-size&#x3D;10,current-page&#x3D;1
```

Comment:
@jalvares-crown @tberey - missing / before the 1? Same issues on some lines below as well if so?

Reply:
@ccs-wattsa  It looks like you have reviewed some old outdated files which don't exist in the latest version of the github repo. A good way to confirm this is to checkout the code and look for the line of code you have commented on. The Github interface can be misleading sometimes.